src/main/scala/karazin/scala/users/group/week1/homework/Homework.scala

```diff
@@ -38,23 +41,68 @@ import scala.annotation.tailrec
 
 object Homework :
 
+  //solution for booleans
   object `Boolean Operators` :
 
-    def not(b: Boolean): Boolean = ???
+    def not(b: Boolean): Boolean =
+      if (b)
+        false
+      else
+        true
+
+    def and(left: Boolean, right: => Boolean): Boolean =

```

Comment:
Please add test to prove that `and`'s behaviour is the same as in Java: when `left` is `false` then right is not calculated.